server/spec/endpoints.test.js

```diff
@@ -0,0 +1,143 @@
+const path = require('path');
+require('dotenv').config({path: path.join(__dirname, '../', '../', '.env')});
+const request = require('supertest');
+
+// setup isolated test environment
+// seed and drop db every time tests are run
+const db = require('../db');
+const { app, server} = require('../server');
+const seeder = require('../db/data/seed');
+const { ReviewData }= require('../db/models/Review');
+
+const TESTDB = process.env.MONGO_URI_TEST;
+
+afterAll(async () => {
+  await seeder.deleteReviews(TESTDB);
+  db.close();
+  server.close();
+});
+
+describe('server', () => {
+ 
+  // seed db once before running tests
+  beforeAll(async() => {
+    await seeder.seedReviews(TESTDB);
+    await db.connect(TESTDB);
+  });
+
+  describe('database', () => {
+    it('should contain 100 records', async (done) => {
+      const reviewData = await ReviewData.find({});
+      expect(reviewData.length)
+        .toBe(100);
+      done();
+    });
+  
+    it('should contain ids 1-100', async (done) => {
+      const records = await ReviewData.find({}).sort('+workspaceId');
+      const track = {};
+      for (let i = 1; i <= 100; i++) {
+        track[i] = 0;
+      }
+      records.forEach((record, i) => {
+        track[record.workspaceId]++;
+      })
+      let pass = true;
+      let count = 0;
+      let keys = Array.from(Object.keys(track));
```

Comment:
I think this `Array.from()`.  `Object.keys` should return an array for you from the keys in the `track` object.

Reply:
yeah it's unnecessary, will remove